manifester/manifester.py

```diff
@@ -188,6 +190,7 @@ def process_subscription_pools(self, subscription_pools, subscription_data):
 
     def trigger_manifest_export(self):
         headers = {"headers": {"Authorization": f"Bearer {self.access_token}"}}
+        limit_exceeded = False
```

Comment:
What's the purpose of this? The label is never referenced.

Reply:
ahhhh it is a variable down on lines 219 and 225. I'd move this just above the loop down there. That should make it more apparent what it is used for

Reply:
I didn't search well enough is the only reason I asked - looked like a new variable in the scope, that never got used.

wouldn't consider its placement blocking, but its a super quick update to make.